setup.py

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env python3
 from setuptools import setup
 
-setup(use_scm_version=True)
+setup()
```

Comment:
Keeping `use_scm_version=True` here can't hurt for backwards compatibility, right?

Reply:
You mean compatibility with older setuptools versions?  We have a requirement in the pyproject.toml that requires a sufficiently recent setuptools:

```
[build-system]
requires = ["setuptools>=42", "setuptools_scm[toml]>=3.4", "wheel"]
```

The `jwst` package also requires `setuptools>=42`.

Reply:
Hmm, yeah.  `setuptools 42` is only a year ago, but I guess this is fine since so many other package requirements are bleeding edge too.  👍